src/utilities/TID300/CobbAngle.js

```diff
@@ -8,9 +8,36 @@ export default class CobbAngle extends TID300Measurement {
             point3,
             point4,
             rAngle,
+            use3DSpatialCoordinates,
             ReferencedSOPSequence
         } = this.props;
 
+        const GraphicData = use3DSpatialCoordinates
```

Comment:
Consider a function that takes a use3DSpatialCoordinates and a list of point items and generates this flattened lists - eg:
toScoord(use3DSpatialCoordinates, point1, point2, point3, point4)
That would clean up the code a fair bit.

Reply:
Done.